code/handlers/logusage.q

```diff
@@ -34,17 +34,21 @@ nextid:{:id+::1}
 // A handle to the log file
 logh:@[value;`logh;0]
 
-// write a query log message
+// write a query log message, direct to stdout if running in finspace
 write:{
-	if[logtodisk;@[neg logh;format x;()]];
-	if[logtomemory; `.usage.usage upsert x];
-	ext[x]}
+    if[.finspace.enabled;@[neg 1;format x;()]];
```

Comment:
I've added an extra layer to the if statement so it'll only check if logtodisk is set if .finspace.enabled is false. Still good to have both for std TorQ but, this will at least make it more obvious.
`$[.finspace.enabled;
	@[neg 1;format x;()];
	if[logtodisk;@[neg logh;format x;()]]];`